Main/src/Main/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,200 @@
+package Main;
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    int water = 400;
+    int milk = 540;
+    int coffeeBeans = 120;
+    int disCups = 9;
+    int money = 550;
+    String buy = "buy";
+    String fill = "fill";
+    String take = "take";
+    String exit = "exit";
+    String remaining = "remaining";
+    String back = "back";
+    int mode = 1;
+
+
+    // method myMethod check coffee machine mode, and call appropriate method
+    public void myMethod(String inputString) {
```

Comment:
I think it would be good idea to change myMethod name

Reply:
@GayaneGulanyan jan @Edvard777  right

Reply:
@nsimonyan 
@Edvard777 
done